source/core/parameters.cc

```diff
@@ -712,7 +712,15 @@ namespace Parameters
         "heat transfer stabilization",
         "false",
         Patterns::Bool(),
-        "Apply discontinuity-capturing directional dissipation stabilization term on heat transfer <true|false>");
+        "Apply discontinuity-capturing directional dissipation stabilization "
+        "term on heat transfer <true|false>");
+
+      prm.declare_entry(
+        "vof dcdd stabilization",
+        "false",
+        Patterns::Bool(),
+        "Apply discontinuity-capturing directional dissipation (dcdd) "
+        "stabilization term on the VOF phase fraction <true|false>");
```

Comment:
In general the DCDD stabilization still leads to good result. I do not think that it should be disabled by default. We have good results for pretty much except the marangoni droplet using it.

Reply:
That's true... Should I keep it enabled or disabled in tests ?

Reply:
Keep it enabled in the tests so they don,t change. You can add one test where it is disabled though